api/v1beta1/cincinnati_types.go

```diff
@@ -50,31 +72,20 @@ const (
 	ConditionRegistryCACertFound conditionsv1.ConditionType = "RegistryCACertFound"
 )
 
-// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-
-// Cincinnati is the Schema for a Cincinnati service.
+// +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:path=cincinnatis,scope=Namespaced
+
+// Cincinnati is the Schema for the cincinnatis API
```

Comment:
[Go style][1]:

>  Comments should begin with the name of the thing being described and end in a period

So I prefer the old `Cincinnati is the Schema for a Cincinnati service.`, even if you want to shuffle order around so you can drop the `kubebuilder` stuff.

[1]: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
I added periods to the `type` comments. These are generated comments BTW.